apps/explorer/src/utils/miscellaneous.ts

```diff
@@ -18,13 +16,10 @@ export function assertNonNull<T>(val: T, message: string): asserts val is NonNul
   }
 }
 
-// eslint-disable-next-line
 function noop(..._args: any[]): void {}
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Is the `no-explicit-any` rule disabled?

Reply:
It was disabled even before.
For example: https://github.com/cowprotocol/cowswap/blob/e4a89e8775324651a1fa85c1ba4ae785e33d8a1f/apps/explorer/src/components/orders/NumbersBreakdown/index.tsx#L48 there is no error